src/wwwroot/js/site.js

```diff
@@ -2,3 +2,26 @@
 // for details on configuring this project to bundle and minify static web assets.
 
 // Write your JavaScript code.
+$(function() {
+    $(".comment-btn").click(function(){
+        addComment();
+    });
+});
+
+
+
+function addComment() {
+    $.ajax ({
+        type: 'POST',
+        url: '/api/v1/CommentApi/AddComment/' + $(".comment-btn").data().movieid,

```

Comment:
Use a variable to get the id before the ajax. Also, try setting the url using
```js
`/api/v1/CommentApi/AddComment/${movieId}` 
```
to concatenate the string